frontend/catalyst/jit.py

```diff
@@ -446,27 +447,33 @@ def __init__(self, fn, compile_options):
 
     @debug_logger
     def __call__(self, *args, **kwargs):
-        # Transparantly call Python function in case of nested QJIT calls.
-        if EvaluationContext.is_tracing():
-            isQNode = isinstance(self.user_function, qml.QNode)
-            if isQNode and self.compile_options.static_argnums:
-                kwargs = {"static_argnums": self.compile_options.static_argnums, **kwargs}
+        # !!! TODO: fix jax.scipy numerical failures with properly fetched lapack calls
+        # As of now, we raise a warning prompting the user to use a callback with catalyst.accelerate()
+        # https://app.shortcut.com/xanaduai/story/70899/find-a-system-to-automatically-create-a-custom-call-library-from-the-one-in-jax
+        # https://github.com/PennyLaneAI/catalyst/issues/753
+        # https://github.com/PennyLaneAI/catalyst/issues/1071
+        with Patcher((jax.scipy.linalg, "expm", JaxLinalgWarner(jax.scipy.linalg.expm))):
```

Comment:
I think this should be moved to where we perform tracing. You should some other things being patched there already.

Reply:
👍

Reply:
Good point. I put them in jit/aot compile's capture. Now after the jaxpr is produced from tracing the patches exit and doesn't linger unnecessarily for mlir.

Reply:
I think putting in the capture method is even better, only needs to be written once :)

Reply:
good point

Reply:
looks good